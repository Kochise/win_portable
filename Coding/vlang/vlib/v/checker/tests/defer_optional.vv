fn opt() ? {}

fn thing() ?string {
	defer {
		opt()
	}
	return 'ok'
}