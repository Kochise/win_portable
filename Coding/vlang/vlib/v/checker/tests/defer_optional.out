vlib/v/checker/tests/defer_optional.vv:5:3: error: opt() returns an option, so it should have an `or {}` block at the end
    3 | fn thing() ?string {
    4 |     defer {
    5 |         opt()
      |         ~~~~~
    6 |     }
    7 |     return 'ok'